src/Input/Select/Select.tsx

```diff
@@ -317,6 +326,8 @@ interface Props extends React.ComponentPropsWithoutRef<typeof SelectInput> {
   noOptionResult?: string;
   removeDropIcon?: boolean;
   removeFloatingLabel?: boolean;
+  error?: string | boolean;
```

Comment:
Same comment - error could be an object as well if we use `FormattedMessage`

Reply:
See comment above.